openjdk/excludes/ProblemList_openjdk8.txt

```diff
@@ -283,10 +283,13 @@ sun/tools/jstatd/TestJstatdExternalRegistry.java	https://github.com/adoptium/aqa
 sun/tools/jstatd/TestJstatdPort.java	https://github.com/adoptium/aqa-tests/issues/115	linux-all
 sun/tools/jstatd/TestJstatdPortAndServer.java	https://github.com/adoptium/aqa-tests/issues/115	linux-all
 sun/tools/jstatd/TestJstatdServer.java	https://github.com/adoptium/aqa-tests/issues/115	linux-all
+sun/tools/native2ascii/Native2AsciiTests.sh https://github.com/adoptium/aqa-tests/issues/2667 windows-all
 sun/misc/Version/Version.java https://bugs.openjdk.java.net/browse/JDK-8244548 generic-all
 com/sun/tools/attach/StartManagementAgent.java	https://github.com/adoptium/aqa-tests/issues/115	generic-all
 tools/pack200/CommandLineTests.java	https://github.com/adoptium/aqa-tests/issues/1051	linux-s390x
 
+sun/tools/native2ascii/Native2AsciiTests.sh
```

Comment:
This seems out of place, what does this line do when read in by jtreg (with no associated issue or platform)?

Reply:
Thanks for spotting this - left in by mistake after copy and paste.  Removed.